### ISSUE TITLE
Couple of Questions

### ISSUE THREAD

**Issue by Ruturajn** [PhyTech-R0/dwm-phyOS#6]
Firstly, I would like to say, this is an excellent piece of work, really commendable. I had a few questions, I wanted to ask.
- Are there any resources for making a custom Arch ISO that I can refer to?
- How are you handling the installation, from the ISO, I mean are you using something like the calamares installer?

I would appreciate it if you could help me with this.

Thanks

**Comment by r0-zero** [comment]
For resources, I would suggest arcolinux youtube channel. There is nothing else I think.
Yes, it uses calamares, with both offline and online installer. If you have more questions, feel free to ask from mail.
arda@phytech.io

**Comment by Ruturajn** [comment]
Hi,
I emailed you sometime back (from: nanotiruturaj@gmail.com), but haven't got a reply (maybe it ended up in spam). Anyway, just wanted to let you know. Also, I wanted to know, how I could set the grub background, I tried doing it, by adding the `GRUB_BACKGROUND` parameter in `airootfs/etc/default/grub`, but it doesn't show up when I boot into the ISO. I have set the parameter to `/home/liveuser/.config/Wallpapers/<some_wallpaper>.png`. I am unable to understand my mistake. I would really appreciate any suggestions regarding this.
Thanks
